docs/user-guide/ze-usage-tips.md

```diff
@@ -3,27 +3,35 @@
 Make the best use of Zowe Explorer with the following tips.
 
 ## Data sets, USS, and jobs persistence settings
+
 You can store any data sets, USS files, or jobs permanently in the **Favorites** tab. Right-click on a data set, USS file, or job and click **Add Favorite**.
 
 ## Identify syntax errors with a syntax highlighter
+
 Zowe Explorer supports a syntax highlighter for data sets. To enhance the experience of using the extension, you can download an extension that highlights syntax.
 
 ## Configure the detected language of a file or data set
 
 You can configure Visual Studio Code to use a specific language for a particular file extension type. This prevents the language for a file or data set opened in Zowe Explorer to be detected incorrectly. To set file associations, see [Add a file extension to a language](https://code.visualstudio.com/docs/languages/overview#_add-a-file-extension-to-a-language).
 
-## Edit a profile
-You can edit existing profiles listed in the **Side Bar** by clicking the profile's **Edit** icon (next to the **Search** icon). The feature lets you modify the information inside your profile.
+## Manage a profile
+
+You can manage existing profiles listed in the **Side Bar** by right clicking on the profile and selecting the desired option from the **Quick Pick**. The feature lets you modify the information included in your profile.
```

Comment:
After right-clicking on a profile, you'll have to select "Manage Profile" in the context menu before the quick pick appears with the available options.

Reply:
may I suggest 
Zowe Explorer offers a multitude of options including profile management by right-clicking the profile name once it's available in the Zowe Explorer MVS, USS, or Jobs tree view.

Reply:
or have it similar to the line below for delete just swapping the wanted action out, it's pretty simple instructions.
`Right-click the profile and select **Manage Profile** to see a list of options in the **Quick Pick**. Choosing the option desired for managing the profile.`